ie-solver/helpers.cpp

```diff
@@ -263,17 +417,33 @@ double stokes_error(const ie_Mat & domain, double id_tol,
       continue;
     }
     Vec2 center(0.5, 0.5);
+
```

Comment:
right, this is now a different solution, bring back the old one in a different function